tests/topotests/lib/common_config.py

```diff
@@ -741,13 +745,19 @@ def start_topology(tgen, daemon=None):
         # Loading empty bgpd.conf file to router, to start the bgp deamon
         router.load_config(TopoRouter.RD_BGP, "{}/{}/bgpd.conf".format(TMPDIR, rname))
 
-        if daemon and 'ospfd' in daemon:
+        if daemon and "ospfd" in daemon:
```

Comment:
Why this change?  

In fact this whole file is littered with changes that have nothing to do with what you are fixing.  Like I said.  If you are cleaning up the style of a file.  Please make those changes in their own commit.  I want to be able to separate out style changes from actual code changes.  I cannot do this with this patchset.

Reply:
https://stackoverflow.com/questions/56011/single-quotes-vs-double-quotes-in-python

Reply:
> https://stackoverflow.com/questions/56011/single-quotes-vs-double-quotes-in-python

Hi Donald ,
 As suggested,  I have made required changes , please look into and let me know if further changes/modifications are needed . Thanks